tools/serverpod_cli/test/integration/analyzer/dart/endpoint_validation/endpoint_method_test.dart

```diff
@@ -139,21 +143,53 @@ class ExampleEndpoint extends Endpoint {
   }
 }
 ''');
-      analyzer = EndpointsAnalyzer(testDirectory);
-      endpointDefinitions = await analyzer.analyze(collector: collector);
-    });
+        analyzer = EndpointsAnalyzer(testDirectory);
+        endpointDefinitions = await analyzer.analyze(collector: collector);
+      });
+      test('then no validation errors are reported.', () {
+        expect(collector.errors, isEmpty);
+      });
 
-    test('then no validation errors are reported.', () {
-      expect(collector.errors, isEmpty);
-    });
+      test('then endpoint definition is created.', () {
+        expect(endpointDefinitions, hasLength(1));
+      });
 
-    test('then endpoint definition is created.', () {
-      expect(endpointDefinitions, hasLength(1));
+      test('then no endpoint method definition is created.', () {
+        var methods = endpointDefinitions.firstOrNull?.methods;
+        expect(methods, isEmpty);
+      });
     });
 
-    test('then no endpoint method definition is created.', () {
-      var methods = endpointDefinitions.firstOrNull?.methods;
-      expect(methods, isEmpty);
+    group(
+        'And the first parameter instead contains a named `Session` parameter when analyzed',
```

Comment:
Suggestion(NIT): Remove capital A and validate that the sentence makes sense when combined with the outer group.

Reply:
[7c32c17](https://github.com/serverpod/serverpod/pull/2597/commits/7c32c179843f7b45b4b4f6b77cb51d76e26568bc)